integration-tools/src/test/java/uk/ac/bbsrc/tgac/miso/integration/test/BoxScanTests.java

```diff
@@ -52,37 +54,27 @@ public void testReferenceConversions() {
   @Test
   public void testCollectionComparison() {
     BoxScan fullScan = getFullScan();
-    String[][] array = fullScan.getBarcodesArray();
-    assertEquals(array[0][0], "11111");
-    assertEquals(array[1][0], "22222");
-    assertEquals(array[0][1], "33333");
-    assertEquals(array[1][1], "44444");
-    
     Map<String, String> map = fullScan.getBarcodesMap();
-    assertEquals(array[0][0], map.get("A01"));
-    assertEquals(array[1][0], map.get("B01"));
-    assertEquals(array[0][1], map.get("A02"));
-    assertEquals(array[1][1], map.get("B02"));
+    assertEquals(map.get(BoxUtils.getPositionString(0, 0)), "11111");
+    assertEquals(map.get(BoxUtils.getPositionString(1, 0)), "22222");
+    assertEquals(map.get(BoxUtils.getPositionString(0, 1)), "33333");
+    assertEquals(map.get(BoxUtils.getPositionString(1, 1)), "44444");
+    assertEquals(map.get(BoxUtils.getPositionString(0, 0)), map.get("A01"));
+    assertEquals(map.get(BoxUtils.getPositionString(1, 0)), map.get("B01"));
+    assertEquals(map.get(BoxUtils.getPositionString(0, 1)), map.get("A02"));
+    assertEquals(map.get(BoxUtils.getPositionString(1, 1)), map.get("B02"));
   }
-  
+
   @Test
   public void testImmutability() {
     BoxScan fullScan = getFullScan();
-    
-    String[][] array = fullScan.getBarcodesArray();
-    assertEquals(array[0][0], "11111");
-    
     Map<String, String> map = fullScan.getBarcodesMap();
-    assertEquals(array[0][0], map.get("A01"));
-    
-    array[0][0] = "changed";
-    map.put("A01", "changed");
-    
-    array = fullScan.getBarcodesArray();
-    assertEquals(array[0][0], "11111");
-    
+    assertEquals(map.get(BoxUtils.getPositionString(0, 0)), "11111");
+    assertEquals(BoxUtils.getPositionString(0, 0), "A01");
+    map.put(BoxUtils.getPositionString(0, 0), "changed");
     map = fullScan.getBarcodesMap();
-    assertEquals(array[0][0], map.get("A01"));
+    assertEquals(map.get(BoxUtils.getPositionString(0, 0)), "11111");
+    assertEquals(map.get(BoxUtils.getPositionString(0, 0)), map.get("A01"));
```

Comment:
`String position = BoxUtils.getPositionString(0, 0);` and use that instead of repeating it 5 times

Though I'm not sure this test will work as written if the map returned actually is immuatable anyway